sdks/ruby/src/lib/hatchet/clients.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+# typed: strict
+
+# Hatchet clients module
+# This module provides access to different client types (REST, gRPC, etc.)
+
+module Hatchet
+  # Client implementations for different protocols
+  module Clients
+    # Load REST client if available
+    begin
+      require_relative "clients/rest"
+    rescue LoadError
+      # REST client not generated yet - this is expected initially
+      # Run `rake api:generate` to generate the REST client
+    end
+
+    # Factory methods for creating pre-configured clients
+    class << self
+      # Create a REST API client using the provided Hatchet configuration
+      #
+      # @param config [Hatchet::Config] The main Hatchet configuration
+      # @return [Hatchet::Clients::Rest::ApiClient] Configured REST API client
+      # @raise [LoadError] if REST client hasn't been generated yet
+      #
+      # @example Create a REST client
+      #   config = Hatchet::Config.new(token: "your-jwt-token")
+      #   rest_client = Hatchet::Clients.rest_client(config)
+      #   workflows_api = Hatchet::Clients::Rest::WorkflowApi.new(rest_client)
+      def rest_client(config)
+        unless rest_available?
+          raise LoadError, "REST client not available. Run `rake api:generate` to generate it from the OpenAPI spec."
```

Comment:
this seems like it should either never happen or be some sort of fatal error, but I'm not sure this is the right error message for a user-facing thing?

Reply:
good flag. i need to sink into this. basically should never hit